weblate/templates/translation.html

```diff
@@ -312,24 +319,78 @@ <h4 class="panel-title">
 {% endif %}
 
 <div class="tab-pane" id="download">
-<form action="{% url 'download_translation' project=object.component.project.slug component=object.component.slug lang=object.language.code %}" method="get" data-persist="download-translation">
-<div class="panel panel-default">
-<div class="panel-heading"><h4 class="panel-title">{% trans "Customize download" %}</h4></div>
-<div class="panel-body">
-{% crispy download_form %}
-</div>
-<div class="panel-footer">
-<input type="submit" value="{% trans "Download" %}" class="btn btn-primary" />
-</div>
-</form>
+  <div class="panel panel-default">
+    <div class="panel-heading">
+      <h4 class="panel-title">
+        {% documentation_icon 'user/files' 'download' right=True %}
+        {% trans "Quick downloads" %}
+      </h4>
+    </div>
+    <table class="table table-listing table-striped">
+      <tr>
+        <th class="number">{{ object.stats.all|intcomma }}</th>
+        <td>
+          {% trans "File in original format as translated in the repository" %}
+        </td>
+        <td class="number">
+          <a class="btn-inline" href="{% url 'download_translation' project=object.component.project.slug component=object.component.slug lang=object.language.code %}">{{ object.component.file_format_name }}</a>
+        </td>
+      </tr>
+      <tr>
+        <th class="number">{{ object.stats.all|intcomma }}</th>
+        <td>
+          {% trans "All strings, converted files enriched with comments. Suitable for translating offline" %}
```

Comment:
```suggestion
          {% trans "All strings, converted files enriched with comments; suitable for translating offline" %}
```
Consistent with the one below at 354